src/holodeck/command.py

```diff
@@ -83,7 +83,7 @@ def add_number_parameters(self, number):
                 A number or list of numbers to add to the parameters.
 
         """
-        if isinstance(number, list) or isinstance(number, tuple):
+        if isinstance(number, (list, tuple, np.ndarray)):
```

Comment:
merge your other pr and you won't need this

Reply:
Not sure why this is still here since the develop branch has identical code. You can remove this change from this branch though and it'll still be in develop

Reply:
Not sure why this is happening either. Will watch after merging.